.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,40 @@
-**IMPORTANT**: PRs without the template will be CLOSED
-
-### Description of your *pull request* and other information
-
 <!--
-
-Explanation of your *pull request* in arbitrary form goes here. Please **make sure the description explains the purpose and effect** of your *pull request* and is worded well enough to be understood. Provide as much **context and examples** as possible
-
+    **IMPORTANT**: PRs without the template will be CLOSED
```

Comment:
This was kept outside comment so it's visible in preview

Reply:
As we are expending that section, I think it makes sense to have this in a comment instead of cluttering up the actual PR. It is the very first thing in the PR Template. You would have to be wilfully ignorant to ignore it, at which point the text being present in the preview won't help things.